src/components/CustomerQuota/CheckoutSuccess/CheckoutSuccessCard.tsx

```diff
@@ -166,8 +166,18 @@ export const CheckoutSuccessCard: FunctionComponent<CheckoutSuccessCard> = ({
     !!quotaResponse?.globalQuota &&
     !!sortedTransactions &&
     sortedTransactions.length > 0 &&
+    /**
+     * We only display global limit messages if there is only one global quota,
+     * since we have not catered for showing global limit messages for multiple
+     * categories.
+     */
+    quotaResponse.globalQuota.length === 1 &&
     !!getProduct(sortedTransactions[0].category)?.quantity.usage;
```

Comment:
Will the latest transaction always be the same policy as what is in `globalQuota`?

Reply:
Ah, you are assuming there is only one policy for this campaign. In this case would a better check be checking that `allProducts` have length 1? Rather than checking for `globalQuota` length

Reply:
Good point, will change this check for both `CheckoutSuccessCard` and `NoQuotaCard`!

Reply:
Ah just to be consistent, should we also change the second condition from 

```
!!getProduct(sortedTransactions[0].category)?.quantity.usage;
```

to 

```
allProducts[0].quantity.usage
```

? 

This way, this computation also doesn't have to wait on fetching past transactions.